setup.py

```diff
@@ -121,8 +121,8 @@
     "importlib_metadata",
     "ipadic>=1.0.0,<2.0",
     "isort>=5.5.4",
-    "jax>=0.4.1,<=0.4.13",
-    "jaxlib>=0.4.1,<=0.4.13",
+    "jax>=0.4.27,<=0.4.38",
+    "jaxlib>=0.4.27,<=0.4.38",
```

Comment:
make sure this is what we want.

@gante already revert this part

https://github.com/huggingface/transformers/commit/949cca4061149050d603e87861b2937e2d9da2c4#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7

Reply:
Mmmm let me check

Reply:
yeah, we want to keep the lower versions :D the newer versions are the ones causing the conflicts